### ISSUE TITLE
How to change domain translation?

### ISSUE THREAD

**Issue by Ronan-Lenor** [whiteoctober/BreadcrumbsBundle#66]
i can't find it in the doc

**Comment by bocharsky-bw** [comment]
The `translation_domain` key in [configuration](https://github.com/whiteoctober/BreadcrumbsBundle#configuration)

**Comment by Ronan-Lenor** [comment]
oh! so there's not dynamic modification like:

$breadcrumbs->addItem( "hello", '/', array(), 'myNewDomain' );

:(

**Comment by bocharsky-bw** [comment]
Also you can pass translation domain as an option to twig render function `wo_breadcrumbs_render`.

**Comment by Ronan-Lenor** [comment]
thanks, you saved my day.

maybe i should pull a request to add a domain option in the other method.
it would be more convenient to use it in a controller i guess

**Comment by bocharsky-bw** [comment]
Yes, you should, PRs are welcome. I think Rich Sage could take a time to review it.

**Comment by Ronan-Lenor** [comment]
yep you are right, let's ask. it's free haha ;)
